ceph-build/config/definitions/ceph-build.yml

```diff
@@ -9,7 +9,7 @@
       - github:
           url: https://github.com/ceph/ceph
     execution-strategy:
-      combination-filter: (ARCH=="x86_64") || (DIST == "trusty" && ARCH == "arm64")
+      combination-filter: (ARCH=="x86_64") || (DIST == "xenial" && ARCH == "arm64")
```

Comment:
No need, but it's a choice: we're only interested in building for xenial right now.  (and none of the rpm stuff is changed at all yet).